lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -1771,13 +1847,15 @@ export class UserAgentApplication {
                         }
                         // Save the token
                         else {
+                            this.logger.verbose("Nonce matches, saving idToken to cache, saving idToken as access token for app");
```

Comment:
The access token thing we do here may be a little confusing to customers, I suggest we don't call this out specifically. It's enough to say we're caching the id token
```suggestion
                            this.logger.verbose("Nonce matches, saving idToken to cache");
```